ush/config.sh.RRFS_AK_dev1

```diff
@@ -31,23 +31,46 @@ DO_DACYCLE="true"
 
 QUILTING="TRUE"
 CCPP_PHYS_SUITE="FV3_HRRR"
-FCST_LEN_HRS="12"
-FCST_LEN_HRS_CYCLES=(12 3 3 12 3 3 12 3 3 12 3 3 12 3 3 12 3 3 12 3 3 12 3 3)
-LBC_SPEC_INTVL_HRS="1"
-EXTRN_MDL_LBCS_OFFSET_HRS="1"
+
+INITIAL_CYCLEDEF="00 00,12 * * 2021 *"
+EXTRN_MDL_ICS_OFFSET_HRS="0"
+BOUNDARY_LEN_HRS="18"
+BOUNDARY_LONG_LEN_HRS="36"
+LBC_SPEC_INTVL_HRS="3"
+EXTRN_MDL_LBCS_OFFSET_HRS="0"
+EXTRN_MDL_LBCS_SEARCH_OFFSET_HRS="0"
+BOUNDARY_CYCLEDEF="00 00,06,12,18 * * 2021 *"
+BOUNDARY_LONG_CYCLEDEF="00 03,09,15,21 * * 2021 *"
+
+FCST_LEN_HRS="24"
+FCST_LEN_HRS_CYCLES=(24 3 3 12 3 3 24 3 3 12 3 3 24 3 3 12 3 3 24 3 3 12 3 3)
```

Comment:
There is a LOT of duplication of information here about the intricate knowledge of the scripts. Can anything be done to help a user to set fewer variables and get to this answer? 

For example, how should one know to choose `BOUNDARY_LONG_LEN_HRS="36"` with `FCST_LEN_HRS_CYCLES=(24 3 3 12 3 3 24 3 3 12 3 3 24 3 3 12 3 3 24 3 3 12 3 3)`. 

If this can't be automated with some coded rules, this needs a lot more documentation here.